### ISSUE TITLE
Low color contrast

### ISSUE THREAD

**Issue by TheoAcker12** [swcarpentry/styles#626]
Low color contrast in code blocks is mentioned in #305. There are a couple other elements that have insufficient contrast:

- Headings in "Caution" blockquotes. WAVE did not detect this due to the use of a linear gradient. Something like `#ff6b75` would provide sufficient contrast without making the background too bright.
- Navbar text. This is not detected by WAVE, because technically the contrast ratio meets AA for large text, even though the navbar text is not large. Changing the color to `#707070` would work, though something like `#545454` would be more readable. The darker it is, however, the less hovering over it changes the color. I am very interested in people's thoughts as to how to handle this.

**Comment by maxim-belkin** [comment]
There is another tool that you can use, it's called tota11y. I think it reports navbar text colors as not having sufficient contrast but is OK with the colors of comments in code blocks.
